### PR TITLE
Share/+Download - Fix convert to domToPng

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

domToPng is failing, this fixes that.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by karimkawambwa** [comment]
> I can't see the attribution HURUmap logo below the chart
> Lorem ipsum dolor sit amet

@kilemensi you have to pass that in, its a must now. I could not figure out how to bundle this asset.

**Comment by kilemensi** [comment]
> @kilemensi you have to pass that in, its a must now. I could not figure out how to bundle this asset.
> 
> The story probably doe not set the asset.

Works on `InsightContainer` so I can assume the asset is there @karimkawambwa ... and on the deployed Storybook, `ChartContainer` story does include the attribution when  `Download` is clicked: https://codeforafrica.github.io/HURUmap-UI/?path=/story/hurumap-ui-chartcontainers-chartcontainer--default

**Comment by karimkawambwa** [comment]
> Works on `InsightContainer` so I can assume the asset is there @karimkawambwa ... and on the deployed Storybook, `ChartContainer` story does include the attribution when `Download` is clicked: https://codeforafrica.github.io/HURUmap-UI/?path=/story/hurumap-ui-chartcontainers-chartcontainer--default

@kilemensi i just pushed to update the stories with the logo

**Comment by kilemensi** [comment]
Issue is here @karimkawambwa: https://github.com/CodeForAfrica/HURUmap-UI/blob/aeb7558db55b5fea97a7a4ebd5e4b750e9af4a79/packages/core/src/ChartContainer.js#L107-L109

The generated className from there ends up like `.makeStyles-Download--only-239` and hence: https://github.com/CodeForAfrica/HURUmap-UI/blob/aeb7558db55b5fea97a7a4ebd5e4b750e9af4a79/packages/core/src/ChartContainer.js#L156-L159

Is never `true`.

File: packages/core/src/ChartContainer.js [hunk 1]
```diff
@@ -151,12 +151,9 @@ function ChartContainer({
   const toPng = () => {
     return domToPng(chartRef.current, {
       filter: node => {
-        if (node.nodeName === 'IMG' && !node.getAttribute('src')) {
-          return false;
-        }
         const { classList } = node;
         if (classList) {
-          if (classList.contains(DOWNLOAD_ONLY_CLASSNAME)) {
+          if (classList.join().includes(DOWNLOAD_ONLY_CLASSNAME)) {
```
Comment (kilemensi) [inline packages/core/src/ChartContainer.js]:
```suggestion
          if (typeof node.className === 'string' && node.className.includes(DOWNLOAD_ONLY_CLASSNAME)){
```

[classList](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList) doesn't support `join` unfortunately (at least in FF).

**Pull request merged**